Sources/RichTextKit/Styles/RichTextStyleToggleGroup.swift

```diff
@@ -60,16 +63,44 @@ public struct RichTextStyleToggleGroup: View {
     private var context: RichTextContext
 
     public var body: some View {
-        ControlGroup {
-            ForEach(styles) {
-                RichTextStyleButton(
-                    style: $0,
-                    buttonStyle: buttonStyle,
+        ZStack {
+            ControlGroup {
+                ForEach(styles) {
+                    RichTextStyleButton(
+                        style: $0,
+                        buttonStyle: buttonStyle,
+                        context: context,
+                        fillVertically: true
+                    )
+                }
+                RichTextLinkButton(
                     context: context,
-                    fillVertically: true
+                    isAlertPresented: $isAlertPresented
                 )
             }
-        }.frame(width: groupWidth)
+            .frame(width: groupWidth)
+            .presentationContainer(
+                style: .sheet,
+                data: context.binding(for: context.link),
+                isPresented: $isAlertPresented) { url in
+                    VStack(spacing: 8) {
+                        TextField("", text: $urlString)
+                            .textFieldStyle(.roundedBorder)
+                            .autocorrectionDisabled(true)
+                           // .textInputAutocapitalization(.never)
```

Comment:
intentionally commented out?

Reply:
macOS doesnt support this, extension will be created later